server/CMakeLists.txt

```diff
@@ -109,6 +110,13 @@ foreach(SCHEMA_FILE IN LISTS IRODS_CONFIGURATION_SCHEMA_FILES)
     COMPONENT ${IRODS_PACKAGE_COMPONENT_SERVER_NAME}
   )
 endforeach()
+# TODO Is this way of updating the ids preferred over using CMake's configure_file() function?
+# If we use input files for the schema files, we can remove these lines and configuration_schemas/update_schema_ids_for_cmake.py.
+# Given that the schemas are maintained with the iRODS source, it would seem we don't need to do this.
+#
+# Q. Is the python script used outside of this location?
+# Q. Given that the configuration_schemas directory contains a README file, should we assume users run the python script directly?
+# Q. Are there any advantages to using this over configure_file()?
```

Comment:
@trel @SwooshyCueb Any thoughts on this?

Reply:
Also, `update_schema_ids_for_cmake.py` is not referenced anywhere else in this repo.

Reply:
Like I said in my other comment, I think ideally we shouldn't be doing this at all. Instead taking the time to switch over to `configure_file`, we should instead put that effort towards eliminating the need for absolute paths in our schema ids.

Reply:
See other comment.